engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -1480,6 +1485,11 @@ class demon_hunter_action_t : public Base
       m *= 1.0 + p()->spec.frailty_debuff->effectN( 4 ).percent() * td( target )->debuffs.frailty->check();
     }
 
+    if ( affected_by.fiery_demise )
```

Comment:
if() condition here needs a td()->fiery_brand->check() I believe?

Reply:
This is leading me down a rabbit hole about the Fiery Brand DoT. Will have an update soon.